docs/reference/setup/configuration.asciidoc

```diff
@@ -71,14 +71,21 @@ path.logs: /var/log/elasticsearch
 
 Environment variables referenced with the `${...}` notation within the
 configuration file will be replaced with the value of the environment
-variable, for instance:
+variable. For example:
 
 [source,yaml]
 --------------------------------------------------
 node.name:    ${HOSTNAME}
 network.host: ${ES_NETWORK_HOST}
 --------------------------------------------------
 
+When specifying values for environment variables, you can only use strings. More complex data structures like lists are not supported. However, settings that accept data structures such as lists also accepted comma-separated strings. For example, you can specify multiple values for the `${HOSTNAME}` variable from the previous example:
```

Comment:
Few things here:

* Be sure to wrap paragraphs to 80 characters or less.
* I typically replace `like` with `such as` for examples, but feel free to ignore that if wanted.
* I think the paragraph is clearer without this sentence:

> However, settings that accept data structures such as lists also accepted comma-separated strings. 

If you decide to keep it, fix the `accepted` bit.

```suggestion
When specifying values for environment variables, you can only use strings. More
complex data structures, such as lists, are not supported.

For example, you can specify multiple values for the `${HOSTNAME}` variable from
the previous example:
```

Reply:
I think this is also not quite right: you can't actually use a comma-separated string to specify multiple values for an environment variable, you're specifying a single value which Elasticsearch will split up if it was expecting a list.

"More complex data structures like lists are not supported" isn't really right, we do support lists of strings this way (but nothing else AFAIK).

Reply:
@DaveCTurner, thank you for the continual feedback. I want to inject clarity in what we're trying to explain! Is this modification closer to what we want to convey?

>Values for environment variables must be simple strings. Use a comma-separated string to provide values that Elasticsearch will parse as a list. For example, Elasticsearch will split the following string into a list of values for the `${HOSTNAME}` environment variable:
```
export HOSTNAME=“host1,host2"
```